examples/js/pmrem/PMREMGenerator.js

```diff
@@ -21,11 +19,13 @@ THREE.PMREMGenerator = ( function () {
 	scene.add( planeMesh );
 	scene.add( camera );
 
-	var PMREMGenerator = function ( sourceTexture, samplesPerLevel, resolution ) {
+	var PMREMGenerator = function ( sourceTexture, params = {} ) {
 
 		this.sourceTexture = sourceTexture;
-		this.resolution = ( resolution !== undefined ) ? resolution : 256; // NODE: 256 is currently hard coded in the glsl code for performance reasons
-		this.samplesPerLevel = ( samplesPerLevel !== undefined ) ? samplesPerLevel : 32;
+		this.sourceResolution = ( params.sourceResolution !== undefined ) ? params.sourceResolution : 512;
+		this.targetResolution = ( params.targetResolution !== undefined ) ? params.targetResolution : 256; // NODE: 256 is currently hard coded in the glsl code for performance reasons
```

Comment:
I think this comment can be removed, and `targetResolution` doesn't need to be a property

Reply:
Why not?   I think some people may want a low resolution if they want to have multiple maps in a large environment so that one can blend between them.

Reply:
To clarify, it doesn't need to be stored on the PMREMGenerator instance and could just be a local variable -- agreed it is valuable to specify the output resolution!